docs/entities-intro.rst

```diff
@@ -178,15 +178,20 @@ Mechanics
 How do I access Entities from my forms?
 ---------------------------------------
 
-First, attach the Entity List you want to access Entities from in your form definition either using `select_one_from_file` or `csv-external`.
+First, attach the Entity List(s) that you want to access Entities from in your form definition using ``select_one_from_file`` or ``csv-external``, as described below. Note that you can attach multiple Entity Lists to a single form via these methods, and they do not need to be listed in the ``entities`` sheet in the XLSForm: only the Entity List in which Entities are created or updated needs to be included in this sheet.
 
-If you want the user to be able to select an Entity from a list, you can use a :ref:`select_one_from_file <select-from-external-dataset>` question with the name of your Entity List followed by `.csv`. For example, if your Entity List is named ``trees``, you would have a ``select_one_from_file trees.csv`` question. 
+* If you want the user to be able to select an Entity from a list, you can use a :ref:`select_one_from_file <select-from-external-dataset>` question with the name of your Entity List followed by ``.csv``. For example, if your Entity List is named ``trees``, you would create a ``select_one_from_file trees.csv`` question.
 
-Everything you know about selects and selects from files apply to attached Entity Lists. For example, you can use an Entity property in a :ref:`choice_filter <cascading-selects>` expression to filter down an Entity List.
+  Everything you know about selects and selects from files applies to attached Entity Lists. For example, you can use an Entity property in a :ref:`choice_filter <cascading-selects>` expression to filter down an Entity List.
 
-If you want to look up Entities using a user-provided value such as a unique ID scanned from a barcode or entered manually, you can attach your Entity List with :ref:`csv-external <form-datasets-attaching-csv>`.
+* If you want to look up Entities using a user-provided value such as a unique ID scanned from a barcode, entered manually, or looked up from another Entity's property, you can attach your Entity List with :ref:`csv-external <form-datasets-attaching-csv>`. For example, if your Entity List is named ``trees``, you would create a form field of type ``csv-external`` with name ``trees``.
 
-Once a specific Entity is identified, you can look up its properties using a :ref:`lookup expression <referencing-values-in-datasets>`. All of this works exactly the same way as it does with CSV form attachments.
+You can access a specific Entity's properties using a :ref:`lookup expression <referencing-values-in-datasets>`. If you've used CSV form attachments or looked values up in a choice list before, looking values up in an Entity List works exactly the same way.
+
+You can attach as many Entity Lists to the same form as you need to support your workflow.
+
+.. image:: /img/entities-intro/multiple-entity-list-attachments.*
```

Comment:
I don't think the image adds much? Might even be confusing because it's not immediately obvious that these are different registration forms and different entity lists.

Reply:
Fair